ui/ui.js

```diff
@@ -326,21 +335,21 @@ shaka.ui.Overlay = class {
         const videoParent = video.parentElement;
         videoParent.replaceChild(container, video);
         container.appendChild(video);
-        let currentCanvas = null;
-        for (const canvas of canvases) {
-          goog.asserts.assert(canvas.tagName.toLowerCase() == 'canvas',
-              'Should be a canvas element!');
-          if (canvas.parentElement == container) {
-            currentCanvas = canvas;
-            break;
-          }
+        let {lcevcCanvas, vrCanvas} =
+            shaka.ui.Overlay.findOrMakeSpecialCanvases_(
+                container, canvases, vrCanvases);
+        if (!lcevcCanvas) {
+          lcevcCanvas = document.createElement('canvas');
+          lcevcCanvas.classList.add('shaka-canvas-container');
+          container.appendChild(lcevcCanvas);
         }
-        if (!currentCanvas) {
-          currentCanvas = document.createElement('canvas');
-          currentCanvas.classList.add('shaka-canvas-container');
-          container.appendChild(currentCanvas);
+        if (!vrCanvas) {
+          vrCanvas = document.createElement('canvas');
+          vrCanvas.classList.add('shaka-vr-canvas-container');
+          container.appendChild(vrCanvas);
         }
```

Comment:
You can include this part inside `findOrMakeSpecialCanvases_`, since this code also repeats in both call-sites.

(Also I just realized I could specify a range of lines when making comments, that's handy.)

Reply:
Done!